src/pudl/extract/epacems.py

```diff
@@ -6,78 +6,86 @@
 import io
 import logging
 from zipfile import ZipFile
+from pathlib import Path
 
 import pandas as pd
 
 from pudl import constants as pc
-from pudl.workspace import datastore as datastore
+from pudl.workspace.datastore import Datastore
+from typing import NamedTuple
 
 logger = logging.getLogger(__name__)
 
 
-class EpaCemsDatastore(datastore.Datastore):
-    """Provide a thin datastore wrapper to ease access to epacems."""
+class EpaCemsPartition(NamedTuple):
+    """Represents EpaCems partition identifying unique resource file."""
+    year: str
```

Comment:
Ha, good catch. In order to ensure that things don't mysteriously break I am converting all values to strings when comparing filters with the parts, in part because there's no readily available typing information that tells you what is expected for individual parts-keys. 

This did bite me because I was running `pudl_datastore --partition year=2018`, the year extracted from commandline flag is a string and the matching subsequently failed.

I guess it makes more sense to use integers here, but in the end, it is irrelevant :)